src-c/native/blake2b.c

```diff
@@ -214,6 +214,13 @@ void digestif_blake2b_finalize( struct blake2b_ctx *ctx, uint8_t *out )
   for( i = 0; i < 8; ++i )
     store64(buffer + sizeof( ctx->h[i] ) * i, ctx->h[i]);
 
-  memcpy( out, buffer, ctx->outlen );
+  secure_zero_memory( out, ctx->outlen * sizeof(uint8_t) );
```

Comment:
this `secure_zero_memory` ([implementation](https://github.com/mirage/digestif/blob/b8ef4e1cb111fd1f406f6c44479c02fff4ca9cb1/src-c/native/bitfn.h#L43-L47)) seems like a homegrown version of `explicit_bzero` ?

Reply:
Yes, but not sure that is available in all targets.